biogeochem/EDPhysiologyMod.F90

```diff
@@ -907,7 +931,7 @@ subroutine phenology_leafonoff(currentSite)
     real(r8) :: store_c_transfer_frac  ! Fraction of storage carbon used to flush leaves
     integer  :: ipft
     real(r8), parameter :: leaf_drop_fraction = 1.0_r8
-
+    real(r8), parameter :: carbon_store_buffer = 0.10_r8
```

Comment:
@rosiealice , would it be most desirable to add this to the parameter file?  We could flag this to be added whenever we have a critical mass of changes to implement.

Reply:
Maybe, but 
1) it's really there to stop us triggering the termination mortality logic, and so is perhaps better thought of as some sort of internal maths?
however:

2) Taking the plant into a very low storage state might in preinciple trigger carbon starvation mortality, even though this is really how deciduous trees are 'supposed' to work. I couldn't really follow the logic of how the target C was calculated in PARTEH well enough to know if there was dispensation for deciduous trees, but irrespective, we don't want to go around killing trees that are really just doing their deciduous things. Then that raises the question of how we -do- get carbon starvation to happen in deciduous trees, and whether those trees should all have a target storage >>1... 

3) Having this as a parameter would allow interrogation of the 'save carbon for a drought' vs. 'put it all in the display pools in the spring' strategies.